### DISCUSSION TITLE
Set Orientation for the Android app using Capacitor only. i.e. no Cordova plugin

### DISCUSSION THREAD

**Discussion by Sampath-Lokuge** [ionic-team/capacitor#4377] [Q&A] [answered]
Hi,

Can you tell me how to do this with the Android app? i.e. using Capacitor to set the orientation for the whole app once.
I saw this feedback here for iOS https://github.com/ionic-team/capacitor/issues/2977#issuecomment-634734685. But how about Android?

**Comment by jcesarmobile** [comment, marked as answer]
In Android apps you can't set the orientation for the whole app, but for the activities in your app.

But since Capacitor has a single activity you can configure it's orientation by adding this to the `activity` tag in `AndroidManifest.xml`

`android:screenOrientation="portrait"` (replace portrait with the desired orientation)

**Reply by Sampath-Lokuge** [reply]
Oh.. Hereafter I  do not need the orientation Cordova plugin at all in most of my apps. Thanks a lot!
